### ISSUE TITLE
Lightweight Multi-View 3D Pose Estimation through Camera-Disentangled Representation

### ISSUE THREAD

**Issue by angechen** [SinaHonari/SinaHonari.github.io#1]
Hello, I'm a student in Northeastern University. I‘m sorry to bother you. I read your paper "Lightweight Multi-View 3D Pose Estimation
through Camera-Disentangled Representation" and I am interested in it. You pretrained a monocular 2d pose estimation network on the COCO dataset, and fine-tune jointly on Human3.6M and MPII datasets. My own 2D pose estimation network can not achieve high accuracy results, so would you please share the 2D pose estimation data trained with multiple datasets. Thanks a lot.
